doc/syntax.rst

```diff
@@ -323,3 +323,75 @@ The advantage of Blackbird templates is that a Blackbird script can encapsulate
 a photonic quantum circuit with free parameters. A library that makes use
 of the Blackbird quantum assembly language (such as Strawberry Fields) can
 dynamically update template parameters without needing to recompile the program.
+
+
+Including subroutines
+---------------------
+
+There may be the case where you have a Blackbird program or template representing
+a circuit primitive that you may want to re-use across multiple Blackbird programs.
+
+This is possible using the ``include`` statement. This has the following syntax:
+
+.. code-block:: python
+
+    include "path/to/filename.xbb"
+
+where the file path is relative to the location of the current Blackbird script.
+A Blackbird script may have multiple includes, and they must all be placed
+after the metadata block, and before the quantum program/variables are defined.
+
+The ``include`` statement allows the external Blackbird program to be used as
+a qquantum subroutine within the existing script. This quantum operation is called
```

Comment:
```suggestion
a subroutine within the existing script. This quantum operation is called
```

Reply:
What clear what "This quantum operation" refers to ("subroutine" is used earlier)